Source/WebCore/rendering/svg/SVGRenderingContext.h

```diff
@@ -74,6 +74,8 @@ class SVGRenderingContext {
     // Support for the buffered-rendering hint.
     bool bufferForeground(RefPtr<ImageBuffer>&);
 
+    static void clear2DRotation(AffineTransform&);
```

Comment:
Please no new stuff in that file - can you keep it in the pattern impl?

Reply:
Sure, looks like the pattern impl may be the only user of clear2DRotation anyway, fixed.